employees/views.py

```diff
@@ -301,20 +305,9 @@ def get_success_url(self) -> str:
         return reverse("custom-report-list", kwargs={"year": self.object.date.year, "month": self.object.date.month})
 
 
-@method_decorator(login_required, name="dispatch")
-@method_decorator(check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name]), name="dispatch")
-class AuthorReportView(MonthNavigationMixin, ProjectsWorkPercentageMixin, DetailView):
-    template_name = "employees/author_report_list.html"
+class BaseAuthorReportView(DetailView, ProjectsWorkPercentageMixin, MonthNavigationMixin):
```

Comment:
Why not adding decorator to base class, as all of the views using it have it?

Reply:
Done